### PR TITLE
PAYARA-2294 Switch to Secure Random

### PR DESCRIPTION
**Pull request by smillidge**

### CODE REVIEW

**Comment by smillidge** [comment]
jenkins test please

**Comment by jerrinot** [comment]
what's the reason for this? see https://www.2uo.de/myths-about-urandom/

**Comment by smillidge** [comment]
Synch from upstream

File: appserver/web/web-core/src/main/java/org/apache/catalina/authenticator/AuthenticatorBase.java [hunk 1]
```diff
@@ -193,7 +193,7 @@
     /**
      * A random number generator to use when generating session identifiers.
      */
-    protected Random random = null;
+    protected SecureRandom random = null;
```
Comment (arjantijms) [inline appserver/web/web-core/src/main/java/org/apache/catalina/authenticator/AuthenticatorBase.java]: Assignment to null is technically not needed

**Pull request merged**